mail-attachment-downloader/README.md

```diff
@@ -0,0 +1,7 @@
+## Mail-Attachment-Downloader
+
+This module contains articles about downloading attachment from email using pop3 protocol.
+
+### Relevant articles:
+
+- [A quick guide to downloading email attachments using Java Mail API](https://www.baeldung.com/)
```

Comment:
We don't add this to the README - it's done after publication.

You've also created a root-level project here. I think we should probably put it somewhere inside an existing module.

https://github.com/eugenp/tutorials/tree/master/libraries-3 seems a good place for me.

Reply:
These changes are now incorporated.

Reply:
Done